interactors/index.js

```diff
@@ -10,5 +10,4 @@ export { default as Select } from './select';
 export { default as Label } from './label';
 export { default as Button } from './button';
 export { default as Pane } from './pane';
-
-export { TextField } from '@bigtest/interactor';
```

Comment:
Do we need to adjust the tests that relied on this lower level version of `TextField`?

Reply:
I don't think so 🤔. I did some grepping, couldn't find this in-use, and the test suite is passing locally.

Reply:
Looks like I might have added it, switched to a different direction and then never removed it? Appears that we don't need to do anything with it.